components/tooltip/index.css

```diff
@@ -14,367 +14,271 @@ governing permissions and limitations under the License.
-  }
-
-  /* tip is vertical at bottom ◁ ▷ */
-  .spectrum-Tooltip--right-bottom &,
-  .spectrum-Tooltip--left-bottom &,
-  .spectrum-Tooltip--start-bottom &,
-  .spectrum-Tooltip--end-bottom & {
-    inset-block-end: var(--mod-tooltip-pointer-corner-spacing, var(--spectrum-tooltip-pointer-corner-spacing));
-  }
-
-  /* tip logical property - Start pointing tip reversed to point toward end ▷ for RTL. */
-  .spectrum-Tooltip--end &,
-  .spectrum-Tooltip--end-top &,
-  .spectrum-Tooltip--end-bottom & {
-    &:dir(rtl) {
-      clip-path: polygon(
-        calc(0% - var(--mod-tooltip-tip-antialiasing-inset, var(--spectrum-tooltip-tip-antialiasing-inset))) 0,
-        calc(0% - var(--mod-tooltip-tip-antialiasing-inset, var(--spectrum-tooltip-tip-antialiasing-inset))) 100%,
-        var(--mod-tooltip-tip-height-percentage, var(--spectrum-tooltip-tip-height-percentage)) 50%
-      );
-      right: auto;
-      left: 100%;
-    }
-  }
-
-  /* tip logical property - End pointing tip reversed to point toward start ◁ for RTL. */
-  .spectrum-Tooltip--start &,
-  .spectrum-Tooltip--start-top &,
-  .spectrum-Tooltip--start-bottom & {
-    &:dir(rtl) {
-      clip-path: polygon(
-        var(--mod-tooltip-tip-height-percentage, var(--spectrum-tooltip-tip-height-percentage)) 50%,
-        calc(100% + var(--mod-tooltip-tip-antialiasing-inset, var(--spectrum-tooltip-tip-antialiasing-inset))) 100%,
-        calc(100% + var(--mod-tooltip-tip-antialiasing-inset, var(--spectrum-tooltip-tip-antialiasing-inset))) 0
-      );
-      left: auto;
-      right: 100%;
-    }
-  }
-}
-
-/****** Icon ******/
-.spectrum-Tooltip-typeIcon {
-  /* subtract tooltip padding from icon start margin */
-  margin-inline-start: calc(var(--mod-tooltip-icon-spacing-inline-start, var(--spectrum-tooltip-icon-spacing-inline-start))
-                            - var(--mod-tooltip-spacing-inline, var(--spectrum-tooltip-spacing-inline)));
-  margin-inline-end: var(--mod-tooltip-icon-spacing-inline-end, var(--spectrum-tooltip-icon-spacing-inline-end));
-  margin-block-start: var(--mod-tooltip-icon-spacing-block-start, var(--spectrum-tooltip-icon-spacing-block-start));
-  inline-size: var(--mod-tooltip-icon-width, var(--spectrum-tooltip-icon-width));
-  block-size: var(--mod-tooltip-icon-height, var(--spectrum-tooltip-icon-height));
-  align-self: flex-start;
-  flex-shrink: 0;
+	position: absolute;
+
+	block-size: var(--mod-tooltip-tip-square-size, var(--spectrum-tooltip-tip-square-size));
+	inline-size: var(--mod-tooltip-tip-square-size, var(--spectrum-tooltip-tip-square-size));
+
+	inset-block-start: calc(100% - (0.5 * var(--mod-tooltip-tip-square-size, var(--spectrum-tooltip-tip-square-size))) - 0.5px);
+	/* stylelint-disable-next-line csstools/use-logical -- intentional use of non-logical property */
+	left: calc(50% - (0.5 * var(--mod-tooltip-tip-square-size, var(--spectrum-tooltip-tip-square-size))));
```

Comment:
I carried this over from PR #2736 until we can get that work into the spectrum-two branch